src/SolverAPI.jl

```diff
@@ -111,6 +111,16 @@ function response(
     res["solve_time_sec"] = Float64(MOI.get(model, MOI.SolveTimeSec()))
 
     result_count = MOI.get(model, MOI.ResultCount())
+
+    try
+        res["relative_gap"] = Float64(MOI.get(model, MOI.RelativeGap()))
```

Comment:
makes a little more sense to do 
```
rel_gap = Float64(MOI.get(model, MOI.RelativeGap()))
``` 
and then only set `res["relative_gap"]` if it's finite.

Reply:
thanks. good point